app/src/main/java/com/simplecity/amp_library/model/SuggestedHeader.java

```diff
@@ -19,11 +19,9 @@ public boolean equals(Object o) {
 
         SuggestedHeader that = (SuggestedHeader) o;
 
-        if (title != null ? !title.equals(that.title) : that.title != null) return false;
-        if (subtitle != null ? !subtitle.equals(that.subtitle) : that.subtitle != null)
-            return false;
-        return playlist != null ? playlist.equals(that.playlist) : that.playlist == null;
-
+        return (title != null ? title.equals(that.title) : that.title == null) &&
```

Comment:
I'd prefer not to make this change. 'Simplifying' the conditional statement generally makes it harder to read for equals() method.